tests/test_twitter_ads_start_date.py

```diff
@@ -0,0 +1,177 @@
+        #  So, skipping those streams from test case.
+        streams_to_test = streams_to_test - {'cards_image_conversation', 'cards_video_conversation', 'cards_image_direct_message',
+                                            'cards_video_direct_message', 'accounts_daily_report', 'campaigns_daily_report', 'accounts',
+                                            'targeting_tv_markets', 'targeting_tv_shows'}
+
+        # running start_date_test for `line_items` and `targeting_criteria` stream
+        expected_stream_1 = {"line_items", "targeting_criteria"}
+        self.run_start_date(expected_stream_1, new_start_date="2022-06-01T00:00:00Z")
+        
+        # running start_date_test for `targeting_events`
+        expected_stream_2 = {'targeting_events'}
+        self.run_start_date(expected_stream_2, new_start_date="2019-06-01T00:00:00Z")
+        
+        # running start_date_test for rest of the streams
+        streams_to_test = streams_to_test - expected_stream_1 - expected_stream_2
+        self.run_start_date(streams_to_test, new_start_date="2022-04-06T00:00:00Z")
+
+    def run_start_date(self, streams_to_test, new_start_date):
+        """
+        Test that the start_date configuration is respected
+        • verify that a sync with a later start date has at least one record synced
+        and fewer records than the 1st sync with a previous start date
+        • verify that each stream has fewer records than the earlier start date sync
+        • verify all data from later start data has bookmark values >= start_date
+        """
+
+        expected_replication_methods = self.expected_replication_method()
+
+        self.start_date_1 = self.get_properties().get('start_date')
+        self.start_date_2 = new_start_date
+        self.start_date = self.start_date_1
+
+        ##########################################################################
+        # First Sync
+        ##########################################################################
+
+        # instantiate connection
+        conn_id_1 = connections.ensure_connection(self)
+
+        # run check mode
+        found_catalogs_1 = self.run_and_verify_check_mode(conn_id_1)
+
+        # table and field selection
+        test_catalogs_1_all_fields = [catalog for catalog in found_catalogs_1
+                                      if catalog.get('tap_stream_id') in streams_to_test]
+        self.perform_and_verify_table_and_field_selection(
+            conn_id_1, test_catalogs_1_all_fields, select_all_fields=True)
+
+        # run initial sync
+        record_count_by_stream_1 = self.run_and_verify_sync(conn_id_1)
+        synced_records_1 = runner.get_records_from_target_output()
+
+        ##########################################################################
+        # Update START DATE Between Syncs
+        ##########################################################################
+   
+        print("REPLICATION START DATE CHANGE: {} ===>>> {} ".format(

```

Comment:
Can we remove `print` statement and use `tap_tester.logger.LOGGER` instead?

Reply:
Replaced print statement with `LOGGER`.